packages/analytics/dashboard-renderer/src/components/BarChartRenderer.vue

```diff
@@ -4,15 +4,16 @@
     :query="query"
     :query-ready="queryReady"
   >
-    <AnalyticsChart
-      :chart-data="data"
-      :chart-options="options"
-      chart-title=""
-      :height="`${height}px`"
-      legend-position="bottom"
-      :show-annotations="chartOptions.showAnnotations"
-      tooltip-title=""
-    />
+    <div class="analytics-chart">
+      <AnalyticsChart
+        :chart-data="data"
+        :chart-options="options"
+        chart-title=""
+        legend-position="bottom"
+        :show-annotations="chartOptions.showAnnotations"
+        tooltip-title=""
```

Comment:
question: what is the purpose of passing in the title attributes as empty strings? I would imagine that's the default?